src/main/java/seedu/eylah/Eylah.java

```diff
@@ -154,6 +166,31 @@ public void run() {
         return new seedu.eylah.expensesplitter.model.ModelManager(new Receipt(), initialData, userPrefs);
     }
 
+    /**
+     * Returns a {@code ModelManager} with the data from {@code storage}'s address book and {@code userPrefs}. <br>
+     * The data from the sample address book will be used instead if {@code storage}'s address book is not found,
+     * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
+     */
+    private seedu.eylah.diettracker.model.Model initModelManagerDiet(FoodBookStorage storage, seedu.eylah.diettracker.model.ReadOnlyUserPrefs userPrefs) {
```

Comment:
We have imported the class for dietTracker's model as Model already, I think we need not be verbose here.